test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -254,7 +254,7 @@
     <!-- In the past, the failure below was caused by an issue where CRIU can't be found on the PATH. -->
     <output type="failure" caseSensitive="yes" regex="no">Could not dump the JVM processes, err=-70</output>
   </test>
-
+-->
```

Comment:
I don't believe you can have nested comments, there are two comments above which will terminate the initial comment.